AzureMonitorAgent/agent.py

```diff
@@ -533,6 +533,11 @@ def enable():
     # Service(s) were successfully configured and started; increment sequence number
     HUtilObject.save_seq()
 
+    # Syslog logging doesn't work on arm64 AlmaLinux until the rsyslog service has been restarted
```

Comment:
As discussed in scrum this should not be necessary with the necessary changes in `common.sh`